helm/reana/values.yaml

```diff
@@ -67,6 +67,8 @@ components:
       REANA_SCHEDULER_REQUEUE_SLEEP: 15
       REANA_USER_EMAIL_CONFIRMATION: true
       REANA_WORKFLOW_SCHEDULING_POLICY: "fifo"
+      REANA_RATELIMIT_GUEST_USER: "20 per second"
+      REANA_RATELIMIT_AUTHENTICATED_USER: "20 per second"
```

Comment:
If these are [the default values in reana-server](https://github.com/reanahub/reana-server/pull/412/files#diff-493aa29a266bfb1026442110e526ad9030f21f4d0e4ea10bdf1ae4c5dafe491dR159-R162), you don't even need to set them here.

Reply:
Done